docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -5092,6 +5092,9 @@ image](#create-an-image) section for more details.
     - **Delay** – Amount of time between updates.
     - **FailureAction** - Action to take if an updated task fails to run, or stops running during the
       update. Values are `continue` and `pause`.
+    - **Monitor** - Amount of time to monitor each updated task for failures.
+    - **MaxFailureRatio** - The fraction of tasks that may fail during an update before the
```

Comment:
I'm somewhat reluctant to document defaults in the API spec because those are left up to the implementation (in this case, SwarmKit). If we were to change one of the defaults in SwarmKit, the API documentation would become misleading. I'm adding the default value for `MaxFailureRatio` because I don't expect we'll change it from 0. But I think it might be better not to indicate a default value for `FailureAction`.